ern-local-cli/src/commands/cauldron/get/dependency.ts

```diff
@@ -16,13 +16,19 @@ export const builder = (argv: Argv) => {
     .coerce('descriptor', d =>
       NativeApplicationDescriptor.fromString(d, { throwIfNotComplete: true })
     )
+    .option('json', {
+      describe: 'json output',
```

Comment:
format output as lines of JSON (see jsonlines.org), please change it all description placeholders. thanks

Reply:
We don't output multiple lines of json here (i.e not one json "record" per line as described in jsonlines.org), but a single json object/array (a single "record"). that being said, that brings a good point, its somewhat easier for third party programs to parse if outputing json object as single line instead of formatted multiline json object. will update the code accordingly.

Reply:
Will also update descriptions to be more meaningful

Reply:
done